src/mdast.rs

```diff
@@ -428,14 +443,19 @@ impl Node {
 
 /// MDX: attribute content.
 #[derive(Clone, Debug, Eq, PartialEq)]
+#[cfg_attr(
+    feature = "json",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(tag = "type", rename = "mdxJsxExpressionAttribute")
+)]
 pub enum AttributeContent {
     /// JSX expression.
     ///
     /// ```markdown
     /// > | <a {...b} />
     ///        ^^^^^^
     /// ```
-    Expression(String, Vec<Stop>),
+    Expression { value: String, stops: Vec<Stop> },
```

Comment:
Should this reuse `AttributeValueExpression`, perhaps then renamed something else?

Reply:
hmm possibly... is there a difference between an `mdxJsxExpressionAttribute` and `mdxJsxAttributeValueExpression`?

Reply:
Well, they all have the same fields, so those are the same.
The only difference is where they occur, and I think the name of the enum itself, already specifies it well!